pkg/networkmanager/firewall.jsx

```diff
@@ -181,6 +181,7 @@ class AddServicesModal extends React.Component {
             custom_udp_ports: [],
             custom_tcp_value: "",
             custom_udp_value: "",
+            zones: firewall.activeZones.size === 1 ? [firewall.defaultZone] : [],
```

Comment:
Please add a comment that this is for a convenient default if there's just one zone.